command/src/main/java/discord4j/command/CommandErrorHandler.java

```diff
@@ -1,23 +1,33 @@
+/*
+ * This file is part of Discord4J.
+ *
+ * Discord4J is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Discord4J is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Discord4J.  If not, see <http://www.gnu.org/licenses/>.
+ */
 package discord4j.command;
 
 import discord4j.core.event.domain.message.MessageCreateEvent;
 import reactor.core.publisher.Mono;
 
-/**
- * An interface called when a {@link CommandException} error is encountered.
- *
- * @see CommandException
- */
+/** An interface called when an error is encountered. */
```

Comment:
I'm fine with this being generalized, but I still think CommandException should exist. As it acts as a way to signal user-error messages which can be consistently handled. And since its end-user error, it's lightweight in that it doesn't make (expensive) stacktraces)